activesupport/test/core_ext/object/blank_test.rb

```diff
@@ -16,7 +16,7 @@ def empty?
     end
   end
 
-  BLANK = [ EmptyTrue.new, nil, false, "", "   ", "  \n\t  \r ", "　", "\u00a0", [], {}, " ".encode("UTF-16LE") ]
+  BLANK = [ EmptyTrue.new, nil, false, "", "   ", "  \n\t  \r ", "　", "\u00a0", "\u200b", "\u200B", "\u200b  ", "  \u200b", "  \u200b  ", "\uFEFF", [], {}, " ".encode("UTF-16LE") ]
```

Comment:
This line is getting rather long, and I'm not sure if we need all of these as separate examples...

```suggestion
  BLANK = [ EmptyTrue.new, nil, false, "", "   ", "  \n\t  \r ", "　", "\u00a0", "\u200B \uFEFF", [], {}, " ".encode("UTF-16LE") ]
```